gammapy/scripts/image_pipe.py

```diff
@@ -301,6 +319,8 @@ def make_images(self, make_background_image=False, bkg_norm=True, spectral_index
         if make_background_image:
             self.maps["bkg"] = total_bkg
             self.maps["exposure"] = total_exposure
+            self.maps["exposure"].unit = obs_image.maps["exposure"].unit
```

Comment:
If you add the unit to the Skymap exposure, you get this error when you save it as a fits with the ```write``` method:
```
ValueError: Illegal value: Unit("cm2 s TeV")
```
I don't know why?